Makefile

```diff
@@ -16,3 +16,27 @@ protogen:
 	protoc \
 		-I sdk -I vendor -I vendor/github.com/gogo/protobuf/protobuf \
 		--gogofaster_out=plugins=grpc:pb sdk/*.proto
+
+integration-test: dummy-build build-lookout
+	$(BIN_PATH)/dummy serve &>/dev/null & \
+	PID=$$!; \
+	$(BIN_PATH)/lookout review ipv4://localhost:10302 | grep "BEGIN RESULT"; \
+	if [ $$? != 0 ] ; then \
+		echo "review test failed"; \
+		kill $$PID; \
+		exit 1; \
+	fi; \
+	$(BIN_PATH)/lookout push ipv4://localhost:10302 | grep "dummy comment for push event"; \
+	if [ $$? != 0 ] ; then \
+		echo "push test failed"; \
+		kill $$PID; \
+		exit 1; \
+	fi; \
+	kill $$PID || true ; \
+
+# target for integration-test. Regular build target puts linux binary to BIN_PATH on MacOS
```

Comment:
I wonder if this should be done as a new rule of Makefile.main, it might be needed in other projects.

Reply:
I don't know any other project where we would need it.
I would suggest to move it to Makefile.main only when there are at least 2 such projects.